packages/fast-components-react-msft/src/action-toggle/action-toggle.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from "react";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { Button, ButtonAppearance } from "../button";
+import {
+    ActionToggleAppearance,
+    ActionToggleHandledProps,
+    ActionToggleProps,
+    ActionToggleUnhandledProps,
+} from "./action-toggle.props";
+import { actionToggleButtonOverrides } from "@microsoft/fast-components-styles-msft";
+import { isNullOrWhiteSpace } from "../../../fast-web-utilities/dist";
```

Comment:
importing from a dist directory will fail in the published package - you need to import this from a named import: `@microsoft/fast-web-utilities`